File: C/3-VendendoJogos.c
#include <stdio.h>

int main() {
    int jogosVendidos;
    scanf("%d", &jogosVendidos);

    double precoJogo = 19.90;
    double vendaTotal = jogosVendidos * precoJogo;
    double bonus = (jogosVendidos / 15) * 0.08 * vendaTotal;
    double salarioTotal = 0.5 * vendaTotal + bonus;

    printf("%.2f\n%.2f\n%.2f\n", vendaTotal, bonus, salarioTotal);

    return 0;
}

File: Haskell/3-VendendoJogos.hs
main :: IO ()
main = do
  gamesSold <- readLn

  let jogoPreco = 19.90
      vendaTotal = fromIntegral jogosVendidos * jogoPreco
      bonus = fromIntegral (jogosVendidos `div` 15) * 0.08 * vendaTotal
      salarioTotal = 0.5 * vendaTotal + bonus

  putStrLn $ formatDouble vendaTotal
  putStrLn $ formatDouble bonus
  putStrLn $ formatDouble salarioTotal

formatDouble :: Double -> String
formatDouble x = printf "%.2f" x
